frontend/src/components/ShootMessageDetails.vue

```diff
@@ -68,9 +68,9 @@ SPDX-License-Identifier: Apache-2.0
             <div v-for="(lastErrorDescription, index) in errorDescriptions" :key="index">
               <v-divider v-if="index > 0" class="my-2"></v-divider>
               <v-alert
-                v-for="({ description, userError, infraAccountError }) in lastErrorDescription.errorCodeObjects" :key="description"
+                v-for="({ description, userError, temporaryError, infraAccountError }) in lastErrorDescription.errorCodeObjects" :key="description"
```

Comment:
```suggestion
                v-for="errorCodeObject in lastErrorDescription.errorCodeObjects" :key="description"
```

Reply:
I would not decompose the errorCodeObject in the loop and forward the object to the icon method.